### PR TITLE
Change the healthcheck protocol to HTTPS in case TLS is enabled

### PR DESCRIPTION
**Pull request by NitishT**

### CODE REVIEW

File: repo/packages/M/minio/8/config.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        "service": {
+            "type": "object",
+            "description": "Service properties for Minio on DC/OS",
+            "properties": {
+                "name": {
+                    "description": "Name of this service instance",
+                    "type": "string",
+                    "default": "minio"
+                },
+                "ssl": {
```
Comment (harshavardhana) [inline repo/packages/M/minio/8/config.json]: ssl is a deprecated protocol i think we should call it TLS?

Comment (NitishT) [inline repo/packages/M/minio/8/config.json]: yes, agreed. Fixed

**Comment by NitishT** [comment]
@ryadav88 any reviews here?

**Comment by harshavardhana** [comment]
Thanks @ryadav88

**Pull request merged**